pkg/cmd/pipeline/start_test.go

```diff
@@ -360,6 +360,109 @@ func TestPipelineStart_ExecuteCommand(t *testing.T) {
 			wantError: true,
 			want:      "no pipelineruns related to pipeline test-pipeline found in namespace ns",
 		},
+		{
+			name: "Dry Run with invalid output",
+			command: []string{"start", "test-pipeline",
+				"-s=svc1",
+				"-r=source=scaffold-git",
+				"-p=pipeline-param=value1",
+				"-l=jemange=desfrites",
+				"-n", "ns",
+				"--dry-run",
+				"-o", "invalid",
+			},
+			namespace: "",
+			input:     cs2,
+			wantError: true,
+			want:      "output format specifed is invalid but must be yaml or json",
+		},
+		{
+			name: "Dry Run with only --dry-run specified",
+			command: []string{"start", "test-pipeline",
+				"-s=svc1",
+				"-r=source=scaffold-git",
+				"-p=pipeline-param=value1",
+				"-l=jemange=desfrites",
+				"-n", "ns",
+				"--dry-run",
+			},
+			namespace: "",
+			input:     cs2,
+			wantError: false,
+			want: `apiVersion: tekton.dev/v1alpha1
```

Comment:
I think these should be golden files, but I think it would be better to refactor these to golden files in a separate pr.

Reply:
Yes they should be golden files :angel: Could be done now or in a follow-up, as you want :wink:

Reply:
Will refactor them all in a separate pr.